### PR TITLE
VHM content moved out of Reference, into Admin

### PR DESCRIPTION
**Pull request by Loquacity**
https://bugzilla.suse.com/show_bug.cgi?id=1139931
https://github.com/SUSE/spacewalk/issues/8239

This resolves the bug, but there's some follow up work to be done:

- [ ] Edit both modules/administration/pages/vhm-vmware.adoc and modules/administration/pages/vhm-file.adoc: https://github.com/SUSE/spacewalk/issues/8255
- [ ] Move other Concept/Task information out of the Reference Guide: https://github.com/SUSE/spacewalk/issues/8256

### CODE REVIEW

File: modules/administration/pages/vhm-file.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+After selecting menu:Create[File-Based] enter a label and URL leading to the location of this file.
+
+
+image::systems_virtual_host_managers_file.png[scaledwidth=80%]
+
+
+.VMWare vCenter Installations without Direct Access
+[NOTE]
+====
+The file-based is not meant to be used with manually crafted files.
```
Comment (keichwa) [inline modules/administration/pages/vhm-file.adoc]:
```suggestion
The file-based option is not meant to be used with manually crafted files.
```

File: modules/administration/pages/vhm-file.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+It only meant to be used with the output of [command]``virtual-host-gatherer`` against some other module.
```
Comment (keichwa) [inline modules/administration/pages/vhm-file.adoc]:
```suggestion
It is only meant to be used with the output of [command]``virtual-host-gatherer`` against some other module.
```

File: modules/administration/pages/vhm-file.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+----
+
+For more information, see the man page on your {productname} server for [command]``virtual-host-gatherer``:
+
+----
+man virtual-host-gatherer
+----
+
+
+The `README` file provided with the package provides background information about the `type` of a hypervisor, etc.:
```
Comment (keichwa) [inline modules/administration/pages/vhm-file.adoc]:
```suggestion
The `README` file of that package provides background information about the `type` of a hypervisor, etc.:
```

File: modules/administration/pages/vhm-file.adoc [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+----
+/usr/share/doc/packages/virtual-host-gatherer/README.md
+----
+
+The man page and the `README` file also contain example configuration files for your reference.
```
Comment (keichwa) [inline modules/administration/pages/vhm-file.adoc]:
```suggestion
The man page and the `README` file also contain example configuration files.
```

File: modules/administration/pages/vhm-vmware.adoc [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+Port::
+Default ESXi API port
+
+Username::
+Assign a username
++
+
+[IMPORTANT]
+====
+Remember that only objects and resources which match a user's defined role will be inventoried.
```
Comment (keichwa) [inline modules/administration/pages/vhm-vmware.adoc]:
```suggestion
Only objects and resources that match a user's defined role will be inventoried.
```

**Comment by Loquacity** [comment]
Thanks @keichwa <3

The whole thing really needs a rewrite, I've opened a different issue for it.

**Comment by keichwa** [comment]
> The whole thing really needs a rewrite, I've opened a different issue for it.

Yes, and you can do this much better than me :)

**Pull request merged**